test/views/html/incomes/HistoricIncomeCalculationViewSpec.scala

```diff
@@ -51,7 +52,7 @@ class HistoricIncomeCalculationViewSpec extends TaiViewSpec {
     }
 
     "have a back link" in {
-      doc must haveBackLink
+      doc.select(".govuk-back-link").text() mustBe messages("tai.back-link.upper")
```

Comment:
should we create a new method for this? Its likely this same test will need to be changed all over

Reply:
oks will do now